lnrpc/rpc.proto

```diff
@@ -1282,15 +1310,35 @@ message PendingChannelsResponse {
 
 message WalletBalanceRequest {
 }
-message WalletBalanceResponse {
+message ChainWalletBalance{
+
+    string chain = 1 [json_name = "chain"];
     /// The balance of the wallet
-    int64 total_balance = 1 [json_name = "total_balance"];
+    int64 total_balance = 2 [json_name = "total_balance"];
 
     /// The confirmed balance of a wallet(with >= 1 confirmations)
-    int64 confirmed_balance = 2 [json_name = "confirmed_balance"];
+    int64 confirmed_balance = 3 [json_name = "confirmed_balance"];
 
     /// The unconfirmed balance of a wallet(with 0 confirmations)
-    int64 unconfirmed_balance = 3 [json_name = "unconfirmed_balance"];
+    int64 unconfirmed_balance = 4 [json_name = "unconfirmed_balance"];
+
+}
+message WalletBalanceResponse {
+
+    repeated ChainWalletBalance chains = 4 [json_name = "chains"];
+}
+
+message ChainBalanceInfo{
+
+    /// Name of chain
+    string chain = 1 [json_name = "chain"];
+
+    /// Sum of channels balances denominated in satoshis
```

Comment:
Same question here on whether we should be using "satoshi" naming for the smallest unit of generic chains.

Reply:
Note that there it is only part of the comment.

My concern is that changing the fields name will bring changes in many files which will put another layer of complication in accepting this PR. Suggest that it will be done later.